azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +195,41 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"dst",
+							}, true),
+						},
+
+						"remote_volume_location": azure.SchemaLocation(),
+
+						"remote_volume_resource_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"replication_schedule": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"_10minutely",
+								"daily",
+								"hourly",
+							}, true),
```

Comment:
could we make this case sensitive
```suggestion
							}, false),
```

Reply:
Changed to case sensitive.